hovercontrol/src/hoverkite.rs

```diff
@@ -146,19 +127,14 @@ impl Hoverkite {
         };
         trace!("Sending command to {:?}: {:?}", side, command);
         if let Some(port) = port {
-            port.write_all(command)?;
+            command.write_to(port)?;
         } else if side == Side::Left {
-            self.forward_command(command)?;
+            // Tell the right side to forward the command to the left side.
+            let port = self.right_port.as_mut().unwrap();
```

Comment:
If both ports are `None` this unwrap will panic. I think it's better to call `self.send_command(Side::Right, ...) to avoid that.

Reply:
I have refactored this into one big `match (side, self.left_port, self.right_port)) {` and added an `error!()` at the bottom if there are no valid ports. Hopefully it is clearer this way. For bonus points, it also lets you do `cargo run -- /dev/null /dev/ttyUSB0` and have it treat the other side as master.

I left the `self.left_last_command_time` handling stuff in its own switch statement, above.